sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -0,0 +1,76 @@
+WITH campaigns AS (
+  SELECT DISTINCT
+    id,
+    name,
+    FORMAT("%s (%s)", name, CAST(id AS string)) AS fenix_compatible_campaign_name
+  FROM
+    `moz-fx-data-bq-fivetran`.google_ads.campaign_history
```

Comment:
Can you pull in `labels` as well? It will require a few joins, but pulling it in as an array is fine (there's `campaign_label_history`, and then `label`). That will help with some campaign analyzing.

Reply:
Chelsea and I decided to leave this for a follow-up